istio-2-mixin/config.libsonnet

```diff
@@ -0,0 +1,32 @@
+{
+  // Static selector to apply to ALL dashboard variables of type query, panel queries, alerts and recording rules.
+  filteringSelector: 'job=~"integrations/istio"',
+  // Used to identify 'group' of instances.
+  groupLabels: ['job', 'cluster'],
+
```

Comment:
@v-zhuravlev are you thinking about having these labels be variables/selectors on the dashboards as well?

Istio is a bit of a different case, where you're interested in how the services/workloads are interacting with each other. And since each pod would correspond with an "instance" selector, I'm not sure how useful an instance and/or a pod variable/selector would be in the dashboards. We already have variables to filter by the things that we care about.

I'm actually not sure if it would even make sense with how the dashboard are structured (the services dashboard is supposed to get info across all pods for a service...so filtering out certain pods seems counter intuitive)

Reply:
Ok, good point! I guess you can skip this then.